src/docs/sdk/performance/time-to-initial-full-display.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Time to Initial Display/Time to Full Display"
+---
+
+We recommend implementing this feature for mobile and desktop SDKs.
+
+The current screen load transactions timings are not reliable to understand the time spent in the screen creation.
+This is because they wait for any span created in the meantime, including any file I/O or network request.
+Time to Initial Display and Time to Full Display are spans added to the screen load transactions,
+which can be used to show more accurate information on the time spent in the screen creation.
+Being spans, they don't impact on user quota.
+
+They start as soon as a new screen is created, as long as the screen load auto-generated transactions are enabled.
```

Comment:
`m`: That is a bit unclear to me. I don't fully understand this sentence.

Reply:
I rephrased it. Is it better now?